django_extensions/management/commands/drop_test_database.py

```diff
@@ -41,10 +41,6 @@ def handle(self, *args, **options):
         """
         Drop test database for this project.
         """
-
-        if args:
-            raise CommandError("reset_db takes no arguments")
```

Comment:
This removal changes behavior, is there a reason for it ? (which cannot be solved otherwise)

Reply:
I actually have no idea how can I pass some arguments using management command so I think that exception will never happen

Reply:
Your probably right.. might be a left over from the old way of parsing command options in Django Commands...